src/lib/server/user.ts

```diff
@@ -0,0 +1,8 @@
+import { getXataClient } from './xata';
+
+const xata = getXataClient();
+
+export async function getAllUsers() {
+	const users = await xata.db.Users.getAll();
+	return users;
+}
```

Comment:
@nrxszvo so thisis the structure i want us to move into group functions relative to their table/model